src/controllers/dev.controller.ts

```diff
@@ -1,10 +1,10 @@
 // src/controllers/dev.controller.ts
 
-import { Request, Response } from 'express';
-import { AtprotoAgent, getActorFeeds } from '../repos/atproto';
-import { saveProfile } from '../repos/profile';
```

Comment:
I think in a different PR we should add a prettier config so that all IDEs follow the same formatting style, mine is just using some default formatting and is changing all single quotes to double quotes 😅

Reply:
Big agree, there are a lot of little things like that that I neglected because it was just me, don't stop calling them out!